docs/source/intro/configure.rst

```diff
@@ -105,8 +105,21 @@ to match the first two numbers from your version.)
 Configuring the ``DATABASES`` setting
 =====================================
 
-After you've set up a project, configure Django's :setting:`DATABASES` setting
-similar to this::
+After you've set up a project, configure Django's :setting:`DATABASES` setting.
+
+If you have a connection string, you can provide it like this::
+
+    DATABASES = {
+        "default": {
+            "ENGINE": "django_mongodb_backend",
+            "HOST": "mongodb+srv://my_user:my_password@cluster0.example.mongodb.net/myDatabase?retryWrites=true&w=majority&tls=false",
```

Comment:
I would remove `myDatabase` from this example since we are targeting Atlas URIs which don't include the database name in the URI.